renderer/renderer.go

```diff
@@ -46,7 +46,7 @@ func New(url string) *Renderer {
 
 func closeResponseBody(ctx context.Context, resp *http.Response) {
 	if err := resp.Body.Close(); err != nil {
-		log.ErrorCtx(ctx, err, log.Data{"message": "error closing http response body"})
+		log.Event(ctx, "error closing http response body", log.Error(err))
 	}
 }
 
```

Comment:
I see that renderer had a Healthcheck() function, but it doesn't have a Checker(). Do we need to implement a Checker function, and add a Check in the Renderer struct, similarly to what you did for other services?

Reply:
Good spot